src/getActors.js

```diff
@@ -0,0 +1,23 @@
+const fetch = require('node-fetch');
+
+function getData(id) {
+  let list = []
+
+  if (!id) {
+    return 'No id'
+  }
+
+  fetch('https://swapi.dev/api/people')
+    .then((response) => response.json())
+    .then((data) => {
+      console.log('list contains:', { data })
+    })
+    .catch(() => {
+      actors = 'Error happend'
+    })
+
+  return actors
+}
+
+const list = getData(1)
+console.log(actors)
```

Comment:
It seems like the 'getData' function will return a promise. You might want to setup the function as an 'async' function and then 'await' the results.
```
async function getData(id)
	...
  try {
    const resp = await fetch('https://swapi.dev/api/people')
    if (resp.ok) {
      actors = await resp.json()
    }
  } catch (err) {
    return Promise.reject('Failed to fetch data!');
  }

  return actors
```
Then when calling this function do something like this:
```
getData(1).then(data => console.log(data))
```